melodic/distribution.yaml

```diff
@@ -2841,6 +2841,17 @@ repositories:
       url: https://github.com/uos/epos2_motor_controller.git
       version: master
     status: maintained
+  er_public_msgs:
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/enabled-robotics/er_public_msgs-release.git
+      version: 1.0.0-1
+    source:
+      type: git
+      url: https://github.com/enabled-robotics/er_public_msgs.git
+      version: 1.0.0
```

Comment:
You cannot use a tag for the source version here.  Please change this to a branch name.